spk/radarr/src/service-setup.sh

```diff
@@ -54,16 +50,15 @@ service_postinst ()
 
 service_preupgrade ()
 {
-    # We have to account for legacy folder under /var/
-    # It should go, after the upgrade, into /.config/
+    # We have to account for legacy folder in the root
+    # It should go, after the upgrade, into /var/.config/
     # The /var/ folder gets automatically copied by service-installer after this
-    # So we need to move our new /.config/ folder to /var/ (if present)
-    if [ -d "${CONFIG_DIR}" ]; then
-        echo "Moving ${CONFIG_DIR} to ${INST_VAR}" >> ${INST_LOG}
-        mv ${CONFIG_DIR} ${INST_VAR}/.config
+    if [ -d "${LEGACY_CONFIG_DIR}" ]; then
```

Comment:
I still do not understand why there is some code to handle `.config` "move" whereas there is no longer "move" as its location has never changed between published package for DSM 5 in synocommunity repository.
May you explain ?

Reply:
OK so I guessed right but that is not my point of view... beta/test packages are "as is"... that is why I avoid to publish when it is not really ready. It is far enough to "support" packages published in repository, not to add complexity to support/track "not ready" packages. Imagine I discard that useless code and let you explain users how to upgrade manually ?

Reply:
I really think you are "pushing" too fast content, before it has been at least reviewed or tested by someone else. You merged a lot of PRs bulk... but be sure I will not publish any of them without reviewing again, and testing locally.

Reply:
> OK so I guessed right but that is not my point of view... beta/test packages are "as is"... that is why I avoid to publish when it is not really ready. It is far enough to "support" packages published in repository, not to add complexity to support/track "not ready" packages. Imagine I discard that useless code and let you explain users how to upgrade manually ?

They will get the update via DSM package center automatically, how do you want to instruct them to upgrade properly? In just the `sonarr` package has more than 400+ downloads among the different versions I released.

Reply:
Your comments here disappoint me. You seem to suggest that it was wrong of me to supply testing packages. That those testing packages were not really ready and that if their setup brakes because you remove this codd, it's kind of my fault. 
While in reality, without testing packages we wouldn't be able to identify problems with this and other  packages. 

And yes, I merge stuff, because I am tired of my code just being left there in those PRs. Yes of course I want you to review and  test them, but when will you actually do that? 
The code just sits there, after I have spend so much time on it. 
I have spend countless hours already since last November, the last 2 weeks even fulltime! And now you make it sound like I just add code without careful consideration. While in fact it was all my careful testing on all platforms that revealed so many previously unknown problems.
I respect your position to handle the publishing and I am glad you do it. So I hope you will also do the job of inspecting and testing my code, not just let it sit there for weeks waiting for.. Well I am not even sure what everything is constantly waiting for here.